packages/ra-auth-google/Readme.md

```diff
@@ -0,0 +1,581 @@
+          element={
+            <div>
+              <OneTapButton />
+              <h1>My Page</h1>
+            </div>
+          }
+        />
+      </CustomRoutes>
+    </Admin>
+  );
+};
+export default App;
+```
+
+### Configure the Google Identity Services library
+
+`initGoogleAuthProvider` accepts all the [parameters](https://developers.google.com/identity/gsi/web/reference/js-reference?hl=en#IdConfiguration) supported by the GIS library.
+
+For example, to change the text of the title and messages in the One Tap prompt, use the [`context`](https://developers.google.com/identity/gsi/web/reference/js-reference?hl=en#context) parameter:
+
+```tsx
+// in src/App.tsx
+import React from "react";
+import { Admin, Resource, Login, CustomRoutes } from "react-admin";
+import { Route } from "react-router-dom";
+import { initGoogleAuthProvider } from "ra-auth-google";
+import dataProvider from "./dataProvider";
+import posts from "./posts";
+
+const App = () => {
+  const { authProvider, LoginButton, OneTapButton } = initGoogleAuthProvider({
+    context: "use",
+  });
+
+  const LoginPage = () => (
+    <Login>
+      <LoginButton />
+    </Login>
+  );
+
+  return (
+    <Admin
+      authProvider={authProvider}
+      dataProvider={dataProvider}
+      title="Example Admin"
+      loginPage={LoginPage}
+    >
+      <Resource name="posts" {...posts} />
+      <CustomRoutes>
+        <Route
+          path="/custom"
+          element={
+            <div>
+              <OneTapButton />
+              <h1>My Page</h1>
+            </div>
+          }
+        />
+      </CustomRoutes>
+    </Admin>
+  );
+};
+export default App;
+```
+
+### Making authenticated requests to the API
+
+`ra-auth-google` includes an `httpClient` that can be used to make authenticated requests to your API. This helper automatically adds the credentials `token` to the request headers.
+
+Here is an example with `ra-data-json-server`:
+
+```tsx
+// in src/App.tsx
+import { initGoogleAuthProvider } from "ra-auth-google";
+import jsonServerProvider from "ra-data-json-server";
+import React from "react";
+import { Admin, Login, Resource } from "react-admin";
+import posts from "./posts";
+
+const App = () => {
+  const { authProvider, LoginButton, httpClient } = initGoogleAuthProvider();
+
+  const dataProvider = jsonServerProvider(
+    "https://jsonplaceholder.typicode.com",
+    httpClient
+  );
+
+  const LoginPage = () => (
+    <Login>
+      <LoginButton />
+    </Login>
+  );
+
+  return (
+    <Admin
+      authProvider={authProvider}
+      dataProvider={dataProvider}
+      title="Example Admin"
+      loginPage={LoginPage}
+    >
+      <Resource name="posts" {...posts} />
+    </Admin>
+  );
+};
+export default App;
+```
+
+### Choosing how to store the token
+
+By default, `ra-auth-google` will store the received token in **localStorage**, under the `"token"` key.
+
+You can choose how to store the token by providing a different `tokenStore`:
```

Comment:
why would one want to do that? Can you give an example?

Reply:
Added following advice from this [review comment](https://github.com/marmelab/ra-auth-google/pull/1#discussion_r1325647318).

It allows to use your own storage method if you don't want to store the token in the localstorage (or want to store it using a different key).

Reply:
I meant give an example *in the readme*

Reply:
Oh I see